sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -1889,18 +1915,23 @@ export class SearchIndexClient {
     analyzeText(indexName: string, options: AnalyzeTextOptions): Promise<AnalyzeResult>;
     // @deprecated
     readonly apiVersion: string;
+    createAlias(alias: SearchIndexerAlias, options?: CreateAliasOptions): Promise<SearchIndexerAlias>;
     createIndex(index: SearchIndex, options?: CreateIndexOptions): Promise<SearchIndex>;
+    createOrUpdateAlias(alias: SearchIndexerAlias, options?: CreateOrUpdateAliasOptions): Promise<SearchIndexerAlias>;
```

Comment:
I know this has changed a few times but at least when I was adding an upsert-like operation to AKV we settled on `set` instead of `createOrUpdate` or `upsert` (so `setAlias` in this case). 

I was also wondering: why is there both `createAlias` and `createOrUpdateAlias` - does `createAlias` throw if the alias already exists? Do we need both?